specs/movie_reserver_spec.rb

```diff
@@ -75,27 +75,36 @@
     end
   end
 
-  xdescribe 'rent_movie' do
-    it 'returns a rental for a successfully rented movie' do
-      # TODO Your Code goes here
-
-
+  # xdescribe 'rent_movie' do
+    xit 'returns a rental for a successfully rented movie' do
+
+      # arrange
+      @rental = [ ]
+      # act
+      # assert
+          ## the rent_movie methods will return
+            
+      @rental.rent_movie(id = 3)
```

Comment:
It might be easier just to give the title of the movie, although you could do it by ID.  However you'll need a `MovieReserver` instance.

You are right these tests below need work.  You might try replicating some of the format of the tests written for you.